x-pack/plugins/remote_clusters/README.md

```diff
@@ -1,5 +1,13 @@
 # Remote Clusters
 
+## Setting up a remote cluster
```

Comment:
Thanks for adding this! I'm still not able to get a remote cluster with a successful connection 🤔. I noticed a few errors in the ES logs related to the license, `...operation due to expired license.`, so might be related to that.

Reply:
Ohhh thats odd, I was able to connect it consistently using this method 🤔